lib/datadog/tracing/contrib/dalli/quantize.rb

```diff
@@ -6,12 +6,32 @@ module Contrib
       module Dalli
         # Quantize contains dalli-specic quantization tools.
         module Quantize
+          # BEGIN BRAZE MODIFICATION
+          GUID_ID_REGEX = /[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}/i
+          BSON_ID_REGEX = /[0-9a-f]{24}/i
+          XXHASH_REGEX = /[0-9a-f]{16}/
+          INTEGER_ID_REGEX = /\d+/
+          # END BRAZE MODIFICATION
           module_function
 
           def format_command(operation, args)
             placeholder = "#{operation} BLOB (OMITTED)"
-            command = [operation, *args].join(' ').strip
+            # BEGIN BRAZE MODIFICATION
+            if operation == :send_multiget
+              command = [operation, *args].join(' ').strip
+            else
+              # all operations except multiget have the key as the first arg
+              command = [operation, args[0]].join(' ').strip
+            end
+            # END BRAZE MODIFICATION
+
             command = Core::Utils.utf8_encode(command, binary: true, placeholder: placeholder)
+            # BEGIN BRAZE MODIFICATION
+            command = command.gsub(GUID_ID_REGEX, "GUID")
+            command = command.gsub(BSON_ID_REGEX, "BSON")
+            command = command.gsub(XXHASH_REGEX, "XXHASH")
+            command = command.gsub(INTEGER_ID_REGEX, "INTEGER")
```

Comment:
this could have negative perf impact, also

Reply:
I don't think this will be too bad, but I'm curious if there are some relatively quick ideas we could do to make this faster. I could smush all of these together into a single expression and just replace it with `ID`